VogtPayroll8/PayrollConsoleReader.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll8
+{
+    class PayrollConsoleReader
+    {
+        #region ReadEmployeeFromConsole
+        /// <summary>
+        /// Reads employee from console
+        /// </summary>
+        /// <returns>An employee</returns>
+        public Employee ReadEmployeeFromConsole()
+        {
+            var empName = "0";
+            var hoursWorked = 0;
+            var payrate = 0m;
+            while (empName == "0")
```

Comment:
Move the while(empName=="0") logic up to the Program.cs while loop.

There should not be a while loop here. You're only returning **one** employee, yet you'll collect 1....n employees here. 

Only the last employee added is actually returned.

Reply:
can't change as it's only for employee name